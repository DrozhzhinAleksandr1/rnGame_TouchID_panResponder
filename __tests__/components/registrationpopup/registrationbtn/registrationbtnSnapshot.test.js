import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import RegistrationBtn from '../../../../app/components/registrationpopup/registrationbtn/registrationbtn';

it('create snapshot RegistrationBtn', () => {
    const renderer = new ShallowRenderer()
    let snapshot = renderer.render(<RegistrationBtn />);
    expect(snapshot).toMatchSnapshot();
});