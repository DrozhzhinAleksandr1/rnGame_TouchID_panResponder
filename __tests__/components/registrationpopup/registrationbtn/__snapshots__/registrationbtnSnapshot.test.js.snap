// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot RegistrationBtn 1`] = `
<TouchableWithoutFeedback
  accessibilityLabel="BtnRegistrationPopupID"
  testID="BtnRegistrationPopupID"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#4BCEA9",
        "height": 56,
        "justifyContent": "center",
        "marginTop": 22,
        "width": 152,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 16,
          "textAlign": "center",
        }
      }
    />
  </Component>
</TouchableWithoutFeedback>
`;
